Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -101,6 +106,67 @@ public class AnalyzeCommand : MultithreadedAnalyzeCommandBase<AnalyzeContext, An
             return skimmers;
         }
 
+        internal static void PushInheritedData(SearchDefinitions definitions, Dictionary<string, string> sharedStrings)
```

Comment:
>PushInheritedData [](start = 29, length = 17)

We should have @jameswinkler provide some unit tests here! Great starter task.

Reply:
Looking into it